src/it/java/com/google/checkstyle/test/chapter5naming/rule526parameternames/ParameterNameTest.java

```diff
@@ -43,36 +49,63 @@ protected String getPath(String fileName) throws IOException {
 
     @BeforeClass
     public static void setConfigurationBuilder() throws CheckstyleException {
-        checkConfig = getCheckConfig("ParameterName");
-        format = checkConfig.getAttribute("format");
+        final List<Configuration> configs = getCheckConfigs("ParameterName");
+
+        Assert.assertEquals(configs.size(), 2);
+
+        privConfig = configs.get(0);
```

Comment:
This can be ignored. `getCheckConfig` calls `DefaultConfiguration.getChildren` which is an array list, not a hashset like other areas. Order is guaranteed here since array list is insertion order and XML parser guarantees order too.